apps/desktop/src/components/ChromeSidebar.svelte

```diff
@@ -0,0 +1,149 @@
+<script lang="ts">
+	import { User } from '$lib/user/user';
+	import { getContextStore } from '@gitbutler/shared/context';
+	import Button from '@gitbutler/ui/Button.svelte';
+	import Icon from '@gitbutler/ui/Icon.svelte';
+	import { goto } from '$app/navigation';
+
+	interface Props {
+		projectId: string;
+	}
+
+	const { projectId }: Props = $props();
+
+	function navigateInProject(target: string) {
+		goto(`/${projectId}/${target}`);
```

Comment:
Could you create a simple `routes.svelte.ts` to mirror `webRoutes.svelte.ts` in e.g. `$lib/routes`? Not sure if we need to call it desktopRoutes since it's not shared like webRoutes.